test/cases/basic.json

```diff
@@ -1,5 +1,6 @@
 {
   "conditions": [],
+  "startPage": "/",
```

Comment:
Ideally, unless we're redirecting back to gov.uk, the start page would always be the first question, which should be the first question defined in the array, and we could get rid of this property?

Reply:
Pages are inserted into the JSON from the designer in the order that they're made. If we did this for france `/your-details` would be our first page. Designer gets them in the right order because it's doing some graphing magic. https://github.com/dagrejs

Reply:
I thought the latest designer did this differently (i.e. had them ordered)? No worries though. We can keeep the property.

Reply:
designer plugin is just given the file. At some point this project was reading pages in order (and/or not the next property properly) so everything was out of order -- even though it looked ok on the designer.